UKHomeOffice/sgmr-service
# Version 2.2.26

Items to review on this version:

## AC / Description of changes
Error reported : The 'SeacraftName' element is invalid - The value 'I'm not on the vessel. I'm based in Scotland' is invalid according to its datatype 'T_SeacraftName' - The actual length is greater than the MaxLength value.

Error reported : The 'PlaceOfBirth' element is invalid - The value 'Z IJsselmeerpolders, Ther Netherlands' is invalid according to its datatype 'T_PlaceOfBirth' - The actual length is greater than the MaxLength value.

## To Test

- run app
- go to pleasure crafts
- go to page 1
- enter a ship name
- > you should not be able to enter more than 35 characters
- try pasting a longer ship name
- > it should truncate at 35 characters

- go to people
- edit a person
- > you should not be able to enter more than 35 characters in place of birth
- try pasting a longer word(s) in the field
- > it should truncate at 35 characters


## Notes

- refactored the character length checker into a component for use elsewhere
- updated the max registration characters to use it
- added it to the ship name
- updated version of RTL user-events so can use the new style user paste
- build includes the version bump to webpack raised by dependabot
- build includes the short term fix to `ais` field so it is not PATCHed as `null`
